### ISSUE TITLE
Equilbrium plot

### ISSUE THREAD

**Issue by JustinJaniszewski** [ubsuny/23-Homework7G4#64]
hello group 4. I am confused about what I should expect for my equilibrium plot. Should it look like the original shape or should it look sort of random? What does yours look like?

**Comment by reshnashrestha** [comment]
Yes you should be comparing that to the actual figures in configurations
